granulate_utils/metrics/yarn.py

```diff
@@ -54,7 +54,12 @@ def _cluster_metrics(self) -> Iterable[Sample]:
 
     def _nodes_metrics(self) -> Iterable[Sample]:
         try:
-            for node in self.rm.nodes(states="RUNNING"):
+            # This are all the statuses that defined as 'active node' in:
+            # isActiveState in hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/api/records/NodeState.java
```

Comment:
Ideally give GH permalinks so one can find it faster + find it even if the file is changed in a future refactor.